code/modules/projectiles/guns/ballistic/launchers.dm

```diff
@@ -146,12 +146,55 @@
 			user.gib()
 		return MANUAL_SUICIDE
 	else
-		user.visible_message("<span class='warning'>[user] sticks [src] in [user.p_their()] mouth, ready to pull the trigger...</span>", \
-			"<span class='userdanger'>You stick [src] in your mouth, ready to pull the trigger...</span>")
+		user.visible_message("<span class='warning'>[user] points [src] at [user.p_their()] head, ready to pull the trigger. This is gonna be messy.</span>", \
+			"<span class='userdanger'>You point [src] at your head, ready to pull the trigger. This is gonna be messy.</span>")
 		sleep(5)
 		shoot_with_empty_chamber(user)
 		sleep(20)
 		user.visible_message("<span class='warning'>[user]'s looks about the room realizing [user.p_theyre()] still there. [user.p_they(TRUE)] proceed to shove [src] further down their throat and choke [user.p_them()]self with it!</span>", \
 			"<span class='userdanger'>You look around after realizing you're still here, then proceed to choke yourself with [src]!</span>")
 		sleep(20)
 		return OXYLOSS
+
+/obj/item/gun/ballistic/rocketlauncher/handle_suicide(mob/living/carbon/human/user, mob/living/carbon/human/target, params, bypass_timer)
```

Comment:
can get rid of the bypass_timer var if you're gonna do this, likewise for the base proc & l144's last true arg.